server/server.js

```diff
@@ -11,6 +11,111 @@ const db = new pg.Pool({
 });
 
 const app = express();
+app.use(express.json());
+
+// GET ENTRIES FROM DATABASE
+app.get('/api/entries', async (req, res) => {
+  try {
+    const sql = `
+    SELECT *
+    FROM "entries"
+    `;
+    const response = await db.query(sql);
+    const allEntries = response.rows;
+    res.json(allEntries);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: 'An unexpected error has occurred.' });
+  }
+});
+
+// POST ENTRY TO DATABASE
+app.post('/api/entries', async (req, res) => {
+  try {
+    const { title, photoUrl, notes } = req.body;
+    if (!title || !photoUrl || !notes) {
+      res.status(400).json({
+        error: 'Invalid information, requires: Title, photoUrl, and Notes.',
+      });
```

Comment:
You need to return here so that you don't execute the SQL statement with invalid data.

Or use the error middleware so you can just throw an exception!